src/DependencyInjection/CompilerPass/SetDoctrineManagerRegistryClassPass.php

```diff
@@ -22,9 +22,9 @@ class SetDoctrineManagerRegistryClassPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
-        if ($container->hasAlias(ManagerRegistry::class)) {
-            $definition = $container->getDefinition('maker.entity_class_generator');
-            $definition->addMethodCall('setMangerRegistryClassName', [ManagerRegistry::class]);
-        }
+//        if ($container->hasAlias(ManagerRegistry::class)) {
+//            $definition = $container->getDefinition('maker.entity_class_generator');
+//            $definition->addMethodCall('setMangerRegistryClassName', [ManagerRegistry::class]);
+//        }
```

Comment:
This isn't needed anymore? Can we delete?

Reply:
yup yup